wordpress-mobile/gutenberg-mobile
# Remove unused JSON asset files

The static JSON file assets are not the sourced by the app. Instead, the data found within the JavaScript bundle itself is utilized. The unnecessary static JSON files should now no longer be exported when bundling the JavaScript thanks to a bug fix in React Native.

- https://github.com/wordpress-mobile/gutenberg-mobile/issues/4329#issuecomment-994876657
- https://github.com/facebook/metro/pull/593

To test: 

<details><summary>Verify the bundle command does not generate JSON assets</summary>

1. Switch to this branch.
1. `npm run bundle`
1. Verify new JSON assets are not generated within the `bundle` directory.

</details>

<details><summary>Verify the correct translation strings display</summary>

1. Switch to this branch.
1. Switch the testing device to a non-English language.
1. Launch the block editor.
1. Verify UI strings display in the correct language.

</details>

PR submission checklist:
